.drone.star

```diff
@@ -887,6 +947,8 @@ def localApiTestPipeline(ctx):
         "federationServer": False,
         "collaborationServiceNeeded": False,
         "extraCollaborationEnvironment": {},
+        "withRemotePhp": [True, False] if ctx.build.event == "cron" else [True],
+        "skipRemotePhpTest": False,
```

Comment:
is the config needed? can we make it work only with `withRemotePhp`?

Reply:
With the current case some suites, for instance `cliCommands`, we don't need to run the builds for both configs and need to be skipped to run on both configs. Since nightly runs with `"withRemotePhp": False` by default, the string `-without-remote.php` will be appended to the pipeline name. This config is added just to make the pipeline name to be same as before.